test/contract.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+import { TIMEOUT } from "./constants";
+import { StarknetContractFactory } from "hardhat/types/runtime";
+import { getPredeployedOZAccount } from "./util";
+import { OpenZeppelinAccount } from "@shardlabs/starknet-hardhat-plugin/dist/src/account";
+import * as fs from "fs";
+import { StarknetContract } from "@shardlabs/starknet-hardhat-plugin/dist/src/types";
+
+describe("StarknetContractFactory", function () {
+    this.timeout(TIMEOUT);
+    let contractFactory: StarknetContractFactory;
+    let account: OpenZeppelinAccount;
+    let contract: StarknetContract;
+    let initial_balance = 25n;
```

Comment:
The linter reports a couple of errors in this file. Do you see them?

Reply:
Yeah, I do... Oops 😮